_docs_integrate/establish-relationships.md

```diff
@@ -112,7 +112,7 @@ If an App user wants to initiate a Relationship with the templator, the App user
 
 After the templator has created a RelationshipTemplate and made it available to another Identity, this Identity can use it to initiate a Relationship with the templator. For this reason, this other Identity is also referred to below as the initiator.
 
-### Check the Initiability of a Relationship
+### Check the Possibility of a Relationship Initiation
```

Comment:
If you want to keep the grammatical structure, use "feasibility" instead of "possibility". However, I still find this wording harder to understand than what @jkoenig134 suggested. If you don't like "establish" and want to keep "initiate", how about "Check if the Relationship can be initiated", instead?

Reply:
Of course I can change "possibility" to "feasibility". 👍 Regarding the other suggestions, I don't like it if we use sentences as titles and I am more a fan of noun sequences in titles. But if it is important to you, I can also change it.

Reply:
Well, I think then it is your personal preference against mine. So all good, no deal breaker to me.